src/test/java/com/amazonaws/services/dynamodbv2/mapper/integration/DynamoDBCryptoIntegrationTestBase.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except
+ * in compliance with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.amazonaws.services.dynamodbv2.mapper.integration;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException;
+import com.amazonaws.services.dynamodbv2.model.AttributeDefinition;
+import com.amazonaws.services.dynamodbv2.model.CreateTableRequest;
+import com.amazonaws.services.dynamodbv2.model.DeleteTableRequest;
+import com.amazonaws.services.dynamodbv2.model.DescribeTableRequest;
+import com.amazonaws.services.dynamodbv2.model.KeySchemaElement;
+import com.amazonaws.services.dynamodbv2.model.KeyType;
+import com.amazonaws.services.dynamodbv2.model.LocalSecondaryIndex;
+import com.amazonaws.services.dynamodbv2.model.Projection;
+import com.amazonaws.services.dynamodbv2.model.ProjectionType;
+import com.amazonaws.services.dynamodbv2.model.ProvisionedThroughput;
+import com.amazonaws.services.dynamodbv2.model.ScalarAttributeType;
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import com.amazonaws.services.dynamodbv2.util.TableUtils;
+import org.testng.annotations.BeforeClass;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class DynamoDBCryptoIntegrationTestBase extends DynamoDBTestBase {
+    protected static final boolean DEBUG = false;
+    protected static final String KEY_NAME = "key";
+    protected static final String TABLE_NAME = "aws-java-sdk-util-crypto";
+
+    protected static long startKey = System.currentTimeMillis();
+
+    protected static final String TABLE_WITH_RANGE_ATTRIBUTE = "aws-java-sdk-range-test-crypto";
+    protected static final String TABLE_WITH_INDEX_RANGE_ATTRIBUTE = "aws-java-sdk-index-range-test-crypto";
+
+    protected static Logger log = Logger.getLogger("DynamoDBCryptoITCaseBase");
+
+    @BeforeClass
+    public static void setUp() throws Exception {
```

Comment:
Please ensure to disable integration tests from CI until we get them all ported to DDB Local.

Reply:
I've ported nearly all of the tests (thanks @mattsb42-aws) to DDB local, and disabled the remaining two for now. I feel OK about not fixing these two immediately, since they appear to explicitly rely on a certain Exception being returned and on performance properties of DynamoDBLocal vs regular DynamoDB. #84 

I've also made a follow-up item to add an option that will allow us to choose whether we target DynamoDB for our tests or DynamoDBLocal. #85